### PR TITLE
chore(SRVKP-4529): tekton results alerts, panels

### PR DESCRIPTION
**Pull request by gabemontero**
Panel screen shot:

results-alerts-panels

rh-pre-commit.version: 2.3.1
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

**Comment by gabemontero** [comment]
@amisstea @redhat-appstudio/konflux-sre PTAL

**Comment by gabemontero** [comment]
for those unaware @redhat-appstudio/konflux-sre ... per https://github.com/redhat-appstudio/o11y/?tab=readme-ov-file#updating-alert-and-recording-rules merging this PR will only affect the recording rules staging environment and the data plane staging environment

File: rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml
Comment (amisstea) [inline rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:153]: How critical is it to filter by the `v1alpha2` version in all these expressions? Could we use just `tekton.results.*` instead so this doesn't break unexpectedly in the event of a Tekton results version bump?

Comment (gabemontero) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:153]: it makes a difference ... unfortunately there is non-results api activity that occurs within the grpc server started within tekton-results-api that we should filter out

I just did a comparison query on prod-rh01 and these diffs showed up:
grpc-with-without-filter

Comment (gabemontero) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:153]: wait ... maybe you meant just remove `v1alpha2` .... just a sec I'll try that

Comment (gabemontero) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml:153]: yeah you did (I read too fast) ... they are the same when I query on prod-rh01 .... I'll update

Comment (gabemontero) [reply rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml]: updated @amisstea sorry for the comment churn ;-)

**Review by amisstea — APPROVED** [review]
LGTM :+1: I'll wait for an approval from @redhat-appstudio/konflux-sre before merging.

**Comment by gabemontero** [comment]
Now, after approvals, we'll have to figure out which order we merge PRs in @amisstea .... my proposal is :

- this PR https://github.com/redhat-appstudio/o11y/pull/342
- https://github.com/redhat-appstudio/o11y/pull/354
- https://github.com/redhat-appstudio/o11y/pull/350
- https://github.com/redhat-appstudio/o11y/pull/349

thanks

**Pull request merged**
